SimulationOutputReport/measure.rb

```diff
@@ -300,25 +300,55 @@ def run(runner, user_arguments)
     return true
   end
 
-  def get_timeseries_size(timeseries_frequency)
+  def get_timestamps(timeseries_frequency)
```

Comment:
Would it make more sense to get these from the [Time table](https://bigladdersoftware.com/epx/docs/9-2/output-details-and-examples/eplusout-sql.html#time-table) (using the appropriate IntervalType) in the sql file?